### ISSUE TITLE
RouterLocaleGuesser allows not allowed locales

### ISSUE THREAD

**Issue by tbopec** [lunetics/LocaleBundle#140]
In the function guessLocale, RouterLocaleGuesser returns true and finishes search in a cases of not allowed locales:
```php
    /**
     * Method that guess the locale based on the Router parameters
     *
     * @param Request $request
     *
     * @return boolean True if locale is detected, false otherwise
     */
    public function guessLocale(Request $request)
    {
        $localeValidator = $this->metaValidator;
        if ($locale = $request->attributes->get('_locale')) {
            if ($localeValidator->isAllowed($locale)) {
                $this->identifiedLocale = $locale;
            }

            return true;
        }

        return false;
    }
```

So, as result, _locale of request is set to not allowed locale

**Comment by JHGitty** [comment]
fixed by #146, right?

**Comment by dbu** [comment]
yeah, i would say so. thanks for cleaning this up @JHGitty !
